iregnet.predictions.R

```diff
@@ -50,13 +48,28 @@ OneFold <- function(testFold.csv){
     })
   }
 
-  #### Stopped HERE
-  
-  
   result.list <- list()
   pred.mat.list <- list()
-  size.i.vec <- seq_along(train.size.vec)
-  ##size.i.vec <- length(train.size.vec)
+  
+  ####
+  
+  fit.list <- list()
+  scale.i.list <- list( estimated = list( init= NA, estimate = TRUE),
+                        fixed = list( init= 1, estimate = FALSE) )
+
+  X.train <- matrix(set.list$train$inputs, nrow(set.list$train$inputs), ncol(set.list$train$inputs))
+  Y.train <-  matrix( set.list$train$outputs , nrow(set.list$train$outputs) , ncol(set.list$train$outputs))
+  
+  for( model.type in c( "gaussian", "logistic", "extreme_value")){
+    for( scale.type in scale.i.list){
+      fit.list[length(fit.list) + 1] <-  cv.iregnet(X.train, Y.train , family = model.type, 
```

Comment:
the loops works until it gets to the extreme_value. then i get this error

`Error in stopifnot_error(paste("family must be one of", paste(names(pfun.list),  : 
  family must be one of gaussian, logistic, exponential
`

Reply:
I tried using iregent (...) instead and it returned this error.

```
Error in dimnames(x) <- dn : 
  length of 'dimnames' [1] not equal to array extent
```
any thoughts?

Reply:
I tried using iregnet( ...., extreme_value,..) instead and it returned this value

```
Error in dimnames(x) <- dn : 
  length of 'dimnames' [1] not equal to array extent
```
any thoughts?

Reply:
first of all you should use `cv.iregnet` (which automatically performs selection of regularization/penalty parameter) and not `iregnet` (which does not perform selection).

for a quick fix I would suggest using only logistic and gaussian (remove extreme value).

also please post a minimal reproducible example as an issue on https://github.com/anujkhare/iregnet/issues . Make sure to include what version of the software you are using, via the packageDescription command, this is what it looks like on my system,
```
> packageDescription("iregnet")
Package: iregnet
Type: Package
Title: Regularized interval regression
Version: 0.1.0.9000
Author: Anuj Khare <khareanuj18@gmail.com>, Toby D Hocking
        <toby.hocking@r-project.org>, Jelle Goeman, Aditya Samantaray
        <aditya.samantaray1@gmail.com>
Maintainer: Anuj Khare <khareanuj18@gmail.com>, Aditya Samantaray
        <aditya.samantaray1@gmail.com>
Description: Interval regression with four types of censoring and
        elastic net regularization.
License: GPL-3
LazyData: TRUE
Suggests: ElemStatLearn, glmnet, testthat, knitr, rmarkdown
LinkingTo: Rcpp
Depends: R (>= 2.10)
Imports: ggplot2, utils, methods, stats, survival, foreach, doParallel,
        data.table, Matrix, namedCapture, penaltyLearning
RoxygenNote: 6.1.1
Remotes: tdhock/penaltyLearning
VignetteBuilder: knitr
RemoteType: github
RemoteHost: api.github.com
RemoteRepo: iregnet
RemoteUsername: anujkhare
RemoteRef: 4d77f047c3a00a5524e1cbe140226417e3aedd92
RemoteSha: 4d77f047c3a00a5524e1cbe140226417e3aedd92
GithubRepo: iregnet
GithubUsername: anujkhare
GithubRef: 4d77f047c3a00a5524e1cbe140226417e3aedd92
GithubSHA1: 4d77f047c3a00a5524e1cbe140226417e3aedd92
NeedsCompilation: yes
Packaged: 2019-07-19 18:40:17 UTC; th798
Built: R 3.6.1; x86_64-w64-mingw32; 2019-07-19 18:40:18 UTC; windows

-- File: C:/Users/th798/R/win-library/3.6/iregnet/Meta/package.rds 
> 
```